__tests__/db.test.js

```diff
@@ -68,6 +68,7 @@ describe("GET /api/articles", () => {
           votes: 0,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          comment_count: 0,
```

Comment:
We shouldn't really go back and change previous tests if possible. This test is a bit brittle as it needed to be changed when you added the comment_count feature. You cold've made this test more robust by using objectContaining or matchObject